pkg/controller/usersignup/usersignup_controller_test.go

```diff
@@ -1962,54 +1962,31 @@ func TestMigrateMur(t *testing.T) {
 		},
 	}
 	mur := newMasterUserRecord(basicNSTemplateTier, "foo", operatorNamespace, "east", "foo")
-	// override values to meet the test requirements.
-	mur.Spec.UserAccounts[0].Spec.NSTemplateSet.Namespaces = []v1alpha1.NSTemplateSetNamespace{
-		{
-			Type:        "dev",
-			Revision:    "123abc1",
-			TemplateRef: "",
-		},
-		{
-			Type:        "stage",
-			Revision:    "123abc2",
-			TemplateRef: "",
-		},
-		{
-			Type:        "extra",
-			Revision:    "123abc3",
-			TemplateRef: "",
-		},
-	}
+
+	// set tier name to be empty
+	mur.Spec.UserAccounts[0].Spec.NSTemplateSet.TierName = ""
+	mur.Spec.UserAccounts[0].Spec.NSLimit = ""
+
 	mur.Spec.UserAccounts[0].Spec.NSTemplateSet.ClusterResources = &toolchainv1alpha1.NSTemplateSetClusterResources{
-		Revision:    "654321b",
-		TemplateRef: "",
+		TemplateRef: "basic-clusterresources-654321b",
 	}
 
 	expectedMur := mur.DeepCopy()
 	expectedMur.Generation = 1
 	expectedMur.ResourceVersion = "1"
-	expectedMur.Spec.UserAccounts[0].Spec.NSTemplateSet.Namespaces = []v1alpha1.NSTemplateSetNamespace{
-		{
-			TemplateRef: "basic-dev-123abc1",
-		},
-		{
-			TemplateRef: "basic-stage-123abc2",
-		},
-		{
-			TemplateRef: "basic-extra-123abc3",
-		},
-	}
+	expectedMur.Spec.UserAccounts[0].Spec.NSTemplateSet.TierName = "basic"
+	expectedMur.Spec.UserAccounts[0].Spec.NSLimit = "default"
+
 	expectedMur.Spec.UserAccounts[0].Spec.NSTemplateSet.ClusterResources = &toolchainv1alpha1.NSTemplateSetClusterResources{
 		TemplateRef: "basic-clusterresources-654321b",
 	}
```

Comment:
This should be:
```go
	// set tier name to be empty
	mur.Spec.UserAccounts[0].Spec.NSTemplateSet = toolchainv1alpha1.NSTemplateSetSpec{}
	mur.Spec.UserAccounts[0].Spec.NSLimit = ""

	expectedMur := mur.DeepCopy()
	expectedMur.Generation = 1
	expectedMur.ResourceVersion = "1"
	expectedMur.Spec.UserAccounts[0].Spec.NSTemplateSet.TierName = "basic"
	expectedMur.Spec.UserAccounts[0].Spec.NSLimit = "default"
	expectedMur.Spec.UserAccounts[0].Spec.NSTemplateSet.Namespaces = []v1alpha1.NSTemplateSetNamespace{
		{
			TemplateRef: "basic-code-123abc1",
		},
		{
			TemplateRef: "basic-dev-123abc2",
		},
		{
			TemplateRef: "basic-stage-123abc3",
		},
	}
	expectedMur.Spec.UserAccounts[0].Spec.NSTemplateSet.ClusterResources = &toolchainv1alpha1.NSTemplateSetClusterResources{
		TemplateRef: "basic-clusterresources-654321b",
	}
```

Reply:
fixed here: d98f4bc8c02bfd5fb7180b49b944aa70a5dc37b2